firstvoices/backend/tests/test_commands/test_rebuild_index.py

```diff
@@ -0,0 +1,61 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from django.core.management import call_command
+
+
+@pytest.mark.django_db
+class TestRebuildIndex:
+    """
+    This class only contains integration tests and does not
+    test the manager.rebuild() itself as that should be covered
+    in its own unit tests.
+    """
+
+    @staticmethod
+    def call_command(*args, **kwargs):
+        call_command(
+            "rebuild_index",
+            *args,
+            **kwargs,
+        )
+
+    def setup_method(self):
+        self.mock_connection = MagicMock()
+        self.mock_manager = MagicMock()
+
+    def test_base_case(self, caplog):
```

Comment:
Is there a need to test individual indexes or just the base case? to my knowledge we haven't needed to use individual indexes though

Reply:
Yes, good question. So the `manager.rebuild()` for the IndexManager class is unit tested separately, but I have added another test for the individual index building, and get the management command to 100% coverage in the latest commit.
The coverage is not visible here since our sonar analysis is broken, but that'll be fixed soon as well.